ifrs17/Import/Importers.ipynb

```diff
@@ -1243,9 +1251,9 @@
                 "\n        await options.TargetDataSource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(null);",
                 "\n        if(ApplicationMessage.HasErrors()) return Activity.Finish().Merge(log);",
                 "\n        ",
-                "\n        // Avoid starting the computation if no best estimate cash flow and actuals has ever been imported ",
-                "\n        if(args.Scenario == null && !(await options.TargetDataSource.Query<RawVariable>().Where(x => x.Partition == targetPartition).Take(1).ToArrayAsync()).Any() &&",
-                "\n                                    !(await options.TargetDataSource.Query<IfrsVariable>().Where(x => x.Partition == targetPartition).Take(1).ToArrayAsync()).Any()) continue;",
+                "\n        // Avoid starting the computation if no best estimate cash flow or actuals has ever been imported ",
+                "\n        if(!(await options.TargetDataSource.Query<RawVariable>().Where(x => x.Partition == defaultPartition).Take(1).ToArrayAsync()).Any() ||",
```

Comment:
checking ifrsVariable is enough. 
In the end when RawVariables are created, also IfrsVariables are.